challenges/visual_referee_challenge.tex

```diff
@@ -1,3 +1,163 @@
+    \caption{\textbf{Pushing Free-kick \textlangle{}color\textrangle{} Team.} Two-handed signal. One arm, vertical with bent elbow and palm facing in the direction of the extended arm. Other arm, extended horizontally in the direction of the half of the field corresponding to the team that is \emph{executing} the Free-kick. That is, left arm extended for the ``Red team'', and right arm extended for the ``Blue team''.}
+\end{figure}
+
+\begin{figure}[ht!]
+    \centering
+    \begin{subfigure}{.33\textwidth}
+        \centering
+        \includegraphics[height=120px]{figs/technical_challenges/full-time-start.png}
+        % \caption{Inner start point of the dynamic movement}
+    \end{subfigure}
+    \begin{subfigure}{.33\textwidth}
+        \centering
+        \includegraphics[height=120px]{figs/technical_challenges/full-time-end.png}
+        % \caption{Outer start point of the dynamic movement}
+    \end{subfigure}
+    \caption{\textbf{Full-Time.} Dynamic two-handed signal. Both arms slowly move symmetrically inward and outwards on a horizontal plane, bending at the elbows. Note, for the purpose of this challenge, the whistle associated with this signal should be a \textit{single} blow, unlike in normal SPL games.}
+\end{figure}
+
+\begin{figure}[ht!]
+    \centering
+    \begin{subfigure}{.33\textwidth}
+        \centering
+        \includegraphics[height=105px]{figs/technical_challenges/player_exchange.png}
+        \vspace*{0.5em}
+        \caption{Rotation around virtual axis to indicate player exchange}
+    \end{subfigure}
+    \hspace*{0.5em}
+    \begin{subfigure}{.6\textwidth}
+        \centering
+        \includegraphics[height=120px]{figs/technical_challenges/kick-in.png}
+        \includegraphics[height=120px]{figs/technical_challenges/kick-in-flipped.png}
+        \caption{Kick-in signal to indicate the team, which is exchanging player. Two possible directions.}
+    \end{subfigure}
+    \caption{\textbf{Player exchange.} Dynamic two-handed signal. Both arms slowly rotate symmetrically around a virtual horizontal axis parallel to the sideline. After three rotations the referee indicates with the kick-in signal the team, which is exchanging player.}
+\end{figure}
+
+\clearpage
+\newpage
+
+\subsection{Challenge evaluation}
+
+A team scores 1 point for every hand-signal that is correctly identified. A team scores an additional 1 point for correctly identifying the team corresponding to the signal (where appropriate: For a correct recognized \textit{Full-Time} signal without announcing a team, the team gets also 1 point for the team color). A team looses 1 point for not or incorrectly identifying a hand-signal (note a team may have a negative final score). The time for the robot to identify a hand-signal is timed (If a robot fails to identify a hand-signal the time for the hand-signal is \qty{15}{\second}. If a robot incorrectly identifies a hand-signal, the time is how long the robot took to provide the incorrect identification).
```

Comment:
This still has the same scoring flaw as last year (for the sided signals):

| | Side wrong | Side correct |
|---|---|---|
| Signal wrong | 0 | 0 |
| Signal correct | 1 | 2 |

i.e. you get partial points for [...], or e.g. if the actual signal is "kick-in red" and you identify "kick-in blue", you get 1 point even though they are *completely different* poses. On the other hand, if the actual signal is "goal kick red" and you identify "corner kick red", you get 0 points, although these signals are very similar.

Therefore, it would make more sense to be

| | Side wrong | Side correct |
|---|---|---|
| Signal wrong | 0 | 1 |
| Signal correct | 0 | 2 |

i.e. you should get partial points for identifying the side but not the signal. 

looses -> loses


Generally, the last two days have reinforced my opinion that things get pushed to master directly, since then I can also immediately fix them instead of trying to explain in a convoluted way what I mean.

Reply:
Done